helpers/tools.ts

```diff
@@ -62,6 +62,13 @@ export const supportedTools: Tool[] = [
     supportedFrameworks: ["fastapi", "express", "nextjs"],
     type: ToolType.LOCAL,
   },
+  {
+    display: "Interpreter",
+    name: "interpreter",
+    dependencies: [],
+    supportedFrameworks: ["fastapi", "express", "nextjs"],
+    type: ToolType.LOCAL,
+  },
```

Comment:
Do we support ToolSpec in TS version? 
E2B interpreter supports executing python and javascript and it requires API key so i think it's better to implement a tool spec for it.
```suggestion
  {
    display: "Code Interpreter",
    name: "e2b.code_interpreter",
    dependencies: [],
    supportedFrameworks: ["fastapi", "express", "nextjs"],
    type: ToolType.LOCAL,
  },
```

Reply:
yes, usually the tool should be configured with E2B's API key. But usually that seems to be in the E2B_API_KEY variable. So for the dev it would be better to edit `E2B_API_KEY` in the env file.
Can we support env variable generation if a specific tool is selected?